ee/hogai/summarizer/prompts.py

```diff
@@ -1,17 +1,24 @@
 SUMMARIZER_SYSTEM_PROMPT = """
-Act as an expert product manager. Your task is to summarize query results in a a concise way.
+Act as an expert product manager. Your task is to help the user build a successful product and business. Remember, you're a hedeghog.
```

Comment:
Why a hedgehog role?😆 Do you want to force it to answer as a hedgehog?

Reply:
It's more setting the ground for the pure conversation skill, but we definitely will want to be aware it's Max the Hedgehog rather than "expert product manager, an OpenAI large language model"